src/test/scala/in/ashwanthkumar/suuchi/store/rocksdb/RocksDbStoreSpec.scala

```diff
@@ -0,0 +1,30 @@
+package in.ashwanthkumar.suuchi.store.rocksdb
+
+import java.io.File
+
+import org.apache.commons.io.FileUtils
+import org.scalatest.{BeforeAndAfter, FlatSpec}
+import org.scalatest.Matchers._
+
+class RocksDbStoreSpec extends FlatSpec with BeforeAndAfter {
+  val ROCKSDB_TEST_LOCATION = "/tmp/suuchi-rocks-test"
```

Comment:
Can we please use `Files.createTempDirectory()` instead of a hard-coded location?